web-app/src/common/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { Game } from 'src/entities/game.entity';
```

Comment:
I know this was done in a previous merge and I didn't see it then but I think all the entity related stuff doesn't belong here because it's very specific to the web application